test/standalone/test-debuglet.js

```diff
@@ -95,6 +96,39 @@ describe(__filename, function(){
     debuglet.start();
   });
 
+  it('should pass source context to api if present', function(done) {
+    fs.writeFileSync('source-context.json', '{ "a": 5 }', 'utf8');
```

Comment:
Would it be possible to make this test state-less? i.e. run it from a directory where a source-context.json already exists? Alternatively, does the config for debug accept a working directory – if so point it to the directory with the source-context.json.

The benefit of stateless tests is that spurious IO issues (stale files from previous runs, or failure to write out these files) don't interfere with the test.